docs/orchestrate/sauce-connect.md

```diff
@@ -0,0 +1,39 @@
+---
+id: sauce-connect
+title: Using Sauce Connect Tunnels with Sauce Orchestrate
+sidebar_label: Using Sauce Connect
+description: Get a Sauce Connect tunnel up and running quickly for your Sauce Orchestrate tests.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Sauce Orchestrate fully integrates with Sauce Connect. If the service or site you are testing is already accessed via Sauce Connect, no change to your code is needed.  
+If your code makes direct calls to any backend service, such as an API or database that is behind a firewall on your private network, follow the steps below to launch a secure trusted connection between your network and the Sauce Orchestrate runner.
+
+
+## What You'll Need
+
+
+## Configure a Sauce Connect Tunnel with access to the management services.
+
+1.  The specific configuration needed for Sauce Orchestrate is setting the `vm-version` to the fixed value listed below and adding the domains of backend services to the [`tunnel-domains`](/dev/cli/sauce-connect-proxy/#--tunnel-domains) list.
```

Comment:
@kmissoumi the link is wrong, what are you trying to link to?

Reply:
https://docs.saucelabs.com/dev/cli/sauce-connect-proxy/#--tunnel-domains